### ISSUE TITLE
Can we compile at build time all the css classes?

### ISSUE THREAD

**Issue by frederikhors** [crswll/clb#6]
Can we compile at build time all the css classes?

**Comment by chasegiunta** [comment]
@frederikhors can you elaborate? Just ensure that all the classes are available in the final build?

If I'm understanding correctly, if you happen to be using Tailwind, as long as you're typing out complete class names in your , you can ensure all are in the final build by passing your .js/.ts files in your purge setting, eg.
```
purge: [
    './src/**/*.{js,ts}',
  ]
  ```

**Comment by frederikhors** [comment]
Nope. I'm talking about something like https://www.npmjs.com/package/babel-plugin-optimize-clsx.

**Comment by crswll** [comment]
It seems like a memo and this plugin would help? I'd have to try it and see. Not really a feature I think that should or could be added here.

**Comment by frederikhors** [comment]
I'm using it with https://github.com/lukeed/clsx and in my Svelte project. It changes code at compile time, reducing runtime overhead, increasing speed.

I think we need something like that.
